tests/integration/components/bs-form/element-test.js

```diff
@@ -148,6 +148,22 @@ module('Integration | Component | bs-form/element', function (hooks) {
     });
   });
 
+  testBS4('controlType "switch" is supported', async function (assert) {
+    await render(hbs`<BsForm::Element @controlType="switch" />`);
```

Comment:
I know that this style is used at many existing tests. But would prefer to invoke it using public API only. Do you mind changing?

```suggestion
    await render(hbs`
      <BsForm as |form|>
        <form.element @controlType="switch" />
      </BsForm>
    `);
```

Reply:
I agree. Again not your fault, as examples like this do exist in the existing code base. Which I would consider bad practice nowadays, and should get refactored over time (e.g. in #1455)